metecho/api/views.py

```diff
@@ -232,6 +303,22 @@ def get_queryset(self):
         repo_ids = self.request.user.repositories.values_list("repo_id", flat=True)
         return self.queryset.filter(repo_id__in=repo_ids)
 
+    def create(self, request):
+        serializer = ProjectCreateSerializer(
+            data=request.data, context=self.get_serializer_context()
+        )
+        serializer.is_valid(raise_exception=True)
+        dependencies = [dep.url for dep in serializer.validated_data["dependencies"]]
+        site_profile = getattr(get_current_site(request), "siteprofile", None)
+        project = serializer.save()
```

Comment:
I think the list of `github_users` passed in is a slightly different shape than what `project.github_users` will expect (missing `name` and `permissions` as least). After creating the new repo, we might want to trigger a `refresh_github_users` call (or do similar logic within the `create_repository` job)?

Reply:
I vaguely remember triggering a refresh somewhere in this branch, but I'll double check

Reply:
Maybe the front end should just submit a list of GH user ids, then let the backend hydrate those or refresh from GH for saving the `project.github_users`?

Reply:
Found it! We are refreshing users after we finalize the repo creation https://github.com/oddbird/Metecho/blob/9434506f0319723955caa92e87f2cf6ab4246a06/metecho/api/models.py#L467-L473

Reply:
Makes sense! Do we even want to save the initially POST-ed `github_users` to `project.github_users` in that case? It seems like maybe the purpose of specifying `github_users` on initial creation is really to add the team on GH, but then we might rely on this `refresh_github_users` to actually populate data on the project? (I'm not confident about this, just thinking "aloud"...)

Reply:
I like that saving the users to the Project  makes them available immediately on the response after posting, and it also removes the need of passing around the list of users as another argument to the job (we can just read them from the Project).